ResidentInformationApi/V1/UseCase/ListContactsUseCase.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using ResidentInformationApi.V1.Boundary.Requests;
+using ResidentInformationApi.V1.Boundary.Responses;
+using ResidentInformationApi.V1.Factories;
+using ResidentInformationApi.V1.Gateways;
+
+namespace ResidentInformationApi.V1.UseCase
+{
+    public class ListContactsUseCase : IListContactsUseCase
+    {
+        private readonly IAcademyInformationGateway _academyGateway;
+        private readonly IHousingInformationGateway _housingGateway;
+        private readonly IMosaicInformationGateway _mosaicGateway;
+
+        public ListContactsUseCase(
+            IAcademyInformationGateway academyGateway,
+            IHousingInformationGateway housingGateway,
+            IMosaicInformationGateway mosaicGateway
+            )
+        {
+            _academyGateway = academyGateway;
+            _housingGateway = housingGateway;
+            _mosaicGateway = mosaicGateway;
+        }
+
+        public async Task<ResidentInformationResponse> Execute(ResidentQueryParam rqp)
+        {
+            var academyClaimants = await _academyGateway.GetClaimantInformation(rqp).ConfigureAwait(true);
+            var housingResidents = await _housingGateway.GetResidentInformation(rqp).ConfigureAwait(true);
+            var mosaicResidents = await _mosaicGateway.GetResidentInformation(rqp).ConfigureAwait(true);
+
+            var academyUrl = Environment.GetEnvironmentVariable("ACADEMYAPI_URL");
```

Comment:
You have renamed these environment variables in the serverless files but they need to also be renamed in the code

Reply:
I've updated them